fesvr/htif.h

```diff
@@ -7,6 +7,7 @@
 #include "syscall.h"
 #include "device.h"
 #include "byteorder.h"
+#include "platform.h"
```

Comment:
To support usage of FESVR as a library, this should be `#include "../riscv/platform.h"`